packages/deployments/contracts/contracts/test/BridgeToken.sol

```diff
@@ -33,6 +33,8 @@ contract BridgeToken is IBridgeToken, OwnableUpgradeable, ERC20 {
 
   function initialize() public override initializer {
     __Ownable_init();
+
+    token.decimals = 18;
```

Comment:
here we need to make sure the `BridgeToken` decimals are the same as the canonical token they are representing, not just that they are always 18.

that means we have to pass that information in plaintext as part of the `BridgeMessage`, instead of only the `detailsHash` so when we deploy the token we can ensure it is set correctly

Reply:
Got it
I will update